### PR TITLE
build: bump version to v12.0.0-rc.0

### PR DESCRIPTION
**Pull request by kyliau**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@angular/devkit-repo",
-  "version": "12.0.0-next.9",
+  "version": "12.0.0-rc.0",
```
Comment (josephperrott) [inline package.json:3]: We are not moving to `rc` yet, we should still be doing a `next` release

**Review by josephperrott — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (kyliau) [reply package.json:3]: This is for the release next week :)

Comment (josephperrott) [reply package.json:3]: Now that I have looked at the calendar... we are only doing feature freeze for one week before moving to rc.  My bad.

**Review by josephperrott — APPROVED** [review]
LGTM

**Pull request merged**